<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lbl_player">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Ход игрока:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_ur_card">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>10</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Твоя карта:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_sum">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Сумма оглашённых карт:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_menu_g">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Назад в меню</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_oglashenie">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>360</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Карта, которую ты огласишь:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cb_value">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>420</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="cb_suit">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>420</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_last_card">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>10</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Предыдущая оглашённая карта:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_ok">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>490</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Закончить ход</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_pass">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>ПАСС</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_ur_image">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>80</y>
      <width>141</width>
      <height>231</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_last_image">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>60</y>
      <width>161</width>
      <height>241</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcd_sum">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_name">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_back_pic">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>571</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_stupid">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>490</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>lbl_back_pic</zorder>
   <zorder>lbl_player</zorder>
   <zorder>lbl_ur_card</zorder>
   <zorder>lbl_sum</zorder>
   <zorder>b_menu_g</zorder>
   <zorder>lbl_oglashenie</zorder>
   <zorder>cb_value</zorder>
   <zorder>cb_suit</zorder>
   <zorder>lbl_last_card</zorder>
   <zorder>b_ok</zorder>
   <zorder>b_pass</zorder>
   <zorder>lbl_ur_image</zorder>
   <zorder>lbl_last_image</zorder>
   <zorder>lcd_sum</zorder>
   <zorder>lbl_name</zorder>
   <zorder>lbl_stupid</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
